source/geometry_model/initial_topography_model/function.cc

```diff
@@ -67,29 +67,19 @@ namespace aspect
           // Now for the vertical component:
           global_point[dim-1] = 0;
 
-          // The point as it is would have to be translated into a different
-          // coordinate system if that was requested in the input file.
-          // This is not currently implemented.
           Assert (coordinate_system == Utilities::Coordinates::CoordinateSystem::cartesian,
-                  ExcNotImplemented());
+                  ExcMessage("While using a box geometry, use cartesian coordinate system."));
```

Comment:
Do you want to say
```suggestion
                  ExcMessage("While using a box geometry, you can only use the cartesian coordinate system."));
```
?